Overtone.Game/OvertoneGame.fs

```diff
@@ -22,10 +23,11 @@ type OvertoneGame(disc: GameDisc, shapesConfig: ShapesConfiguration, windowConfi
         this.GraphicsDevice,
         (disc.ReadFile "THING1/FLOAT.EXE").Result // TODO[#35]: Show a loader at start instead of .Result
     )
-    let scene = lazy MenuScene(lifetime, this.GraphicsDevice, textureManager.Value, windowConfig)
+    let scene = lazy MainScene(lifetime, this.GraphicsDevice, textureManager.Value, windowConfig)
 
     override this.Initialize() =
         this.Window.Title <- "Overtone"
+        scene.Value.changeSceneId 0
```

Comment:
_(Minor)_ I'd prefer to have this `0` in some sort of a named constant. This is the main menu scene, I believe?